.github/workflows/update-deps.yml

```diff
@@ -75,3 +75,13 @@ jobs:
       changelog-entry: false
     secrets:
       api-token: ${{ secrets.CI_DEPLOY_KEY }}
+
+  maestro:
+    uses: getsentry/github-workflows/.github/workflows/updater.yml@v2
+    with:
+      path: scripts/update-maestro.sh
+      name: Maestro
+      pattern: '^v[0-9.]+$' # only match non-preview versions
```

Comment:
[The Maestro version tags](https://github.com/mobile-dev-inc/Maestro/tags) are [similar with RN](https://github.com/facebook/react-native/tags) starting with `v`. A potential issue is that this might not be used consistently (e.g. `v0.2.5` 🤔) and there are also more than one tags on the release (e.g. `v1.40.0` and `cli-1.40.0`). Looking at the [dependency updater script](https://github.com/getsentry/github-workflows/blob/cd1c852719aaf272f2730a2df1146dba80412ee3/updater/scripts/update-dependency.ps1) I think this is the best approach for now.